lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -17,6 +17,7 @@
 #include <glog/logging.h>
 #include <grpcpp/impl/codegen/completion_queue.h>
 #include <lte/protos/mconfig/mconfigs.pb.h>
```

Comment:
My understanding is that `<lte/protos/mconfig/mconfigs.pb.h>`and `<orc8r/protos/mconfig/mconfigs.pb.h>` are headers of the local project. Then instead of `<...>`, you should use `"..."` here  because the preprocessor searches only along the specified path and ignores standard libraries.
(see https://docs.microsoft.com/en-us/cpp/preprocessor/hash-include-directive-c-cpp?view=msvc-170)